### DISCUSSION TITLE
[#995503] /execute can bypass locked commands

### DISCUSSION THREAD

**Discussion by LateLag** [Mojang/minecraft-editor#87] [Bugs]
**Steps to Reproduce:**

1. Open chat and type in /gamemode survival
2. Now try /execute run gamemode survival

**Observed Results:**
While using the raw /gamemode fails, "disguising" it behind /execute works without an issue.

**Expected Results:**
The game always recognizes set limitations no matter how the command is input.

**Comment by minecraftchloe** [comment]
Great catch!!

**Comment by rpoerschke** [comment]
Fixed this issue in 0.3.3.  Thank you!
